app/controllers/merchant/items_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Merchant::ItemsController < ApplicationController
+  before_action :require_merchant
+
+  def index
+    @items = current_user.merchant.items
+  end
+
+  def deactivate
+    merchant = current_user.merchant
```

Comment:
Oh snap! I finally understand why we're calling `current_user.merchant` instead of just `current_user`! We want the merchant entity, not the employee of the entity.